libdnf/module/module_sack_impl.hpp

```diff
@@ -36,6 +37,25 @@ class ModuleSack::Impl {
 
     const std::vector<std::unique_ptr<ModuleItem>> & get_modules();
 
+    /// Apply modular filtering to package set. For proper functionality it requires loaded of all repositories and
+    /// resolve of active modules (modular solver)
```

Comment:
For proper functionality it requires loaded of all repositories  and resolve of active modules (modular solver)
-> For proper functionality, all repositories must be loaded and active modules must be resolved (modular solver).